### PR TITLE
Add some more assert context.

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

File: Foundation/GTMSQLite.m [hunk 1]
```diff
@@ -531,7 +531,8 @@ static void UpperLower8(sqlite3_context *context, int argc, sqlite3_value **argv
   }
 
   _GTMDevAssert(userArgs->textRep == SQLITE_UTF8,
-                @"Received non UTF8 encoding in UpperLower8");
+                @"Received non UTF8 encoding in UpperLower8, %d",
```
Comment (dmaclach) [inline Foundation/GTMSQLite.m:534]: nit, you divide your format string with a "," here and a colon above. Standardize?

File: Foundation/GTMSQLite.m [hunk 2]
```diff
@@ -725,7 +726,8 @@ static void UpperLower16(sqlite3_context *context,
       // we might as well use the preferred encoding of the original call.
       _GTMDevAssert(userArgs->textRep == SQLITE_UTF16BE ||
                     userArgs->textRep == SQLITE_UTF16LE,
-                    @"Received non UTF8 encoding in UpperLower8");
+                    @"Received non UTF16 encoding in UpperLower16, %d",
```
Comment (dmaclach) [inline Foundation/GTMSQLite.m:729]: nit, you divide your format string with a "," here and a colon above. Standardize?

Comment (thomasvl) [reply Foundation/GTMSQLite.m]: done

File: Foundation/GTMSQLite.m (hunk 1 above)
Comment (thomasvl) [reply Foundation/GTMSQLite.m]: done

**Pull request merged**